src/components/common/nav-menu/index.js

```diff
@@ -1,13 +1,32 @@
 import React, { PureComponent } from 'react';
 import { Navbar, Nav, NavItem } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 import { ROUTES } from '../../../constants';
 
 class NavMenu extends PureComponent {
+  // eslint-disable-next-line class-methods-use-this
+  logOut() {
+    localStorage.removeItem('OS_AUTH_TOKEN');
+  }
+
```

Comment:
Like the authentication check, the log out function should not be defined in a component. Move this to an authentication helper/util as you will likely reuse the function again in a different component or module.